palindrom

```diff
@@ -0,0 +1,34 @@
+function pal(start , end , thisword){
+
+   
+    if( thisword[start] != thisword[end] ){
+        console.log(thisword[start]);
+        console.log(thisword[end]);
+        return false;
+    }
+   
+    if(start != end){
+        pal(start+1 , end-1 , thisword);
+        console.log("plaindrom");
+    }
+    
+    return true;
+}
+
+function isPalindromeFunction(str)
+{
+    let l = str.length
```

Comment:
Always use descriptive names for naming identifiers (variables, functions, etc.). Also this variable doesn't change, so it's better to declare it with `const`.

```js
const length = str.length;
```